extensions/positron-assistant/src/utils.ts

```diff
@@ -196,3 +273,16 @@ export function hasNonEmptyContent(message: vscode.LanguageModelChatMessage2): b
 		return true;
 	});
 }
+
+// This type definition is from Vercel AI, but the type is not exported.
```

Comment:
I wasn't sure where to put this type -- it's in `ai/index.d.ts`, but it's not exported.